backend/src/org/akvo/lumen/import/csv.clj

```diff
@@ -142,14 +142,16 @@
           copy-sql (get-copy-sql temp-table n-cols headers? encoding)]
       (jdbc/execute! tenant-conn [(get-create-table-sql temp-table n-cols "text" true)])
       (jdbc/execute! tenant-conn [(get-create-table-sql table-name n-cols "jsonb" false)])
-      (with-open [conn (-> tenant-conn :datasource .getConnection)]
-        (let [copy-manager (.getCopyAPI (.unwrap conn PGConnection))]
-          (.copyIn copy-manager copy-sql (io/input-stream path))))
+      (with-open [conn (-> tenant-conn :datasource .getConnection)
+                  input-stream (-> (io/input-stream path)
+                                   import/unix-line-ending-input-stream)]
+        (let [copy-manager (.. conn (unwrap PGConnection) (getCopyAPI))]
```

Comment:
I would argue that `.` and `..` are lower level constructs than e.g. `.unwrap`, `.getCopyAPI`, `.copyIn` (as on the next line).

I don't have strong feelings about this but we've been mostly using the sugar versions for java interop,  `(.theMethod theObject)` instead of `(. theObject theMethod)`.